vividus-tests/src/main/resources/story/integration/Aliases.story

```diff
@@ -0,0 +1,12 @@
+Description: Integration tests for aliases.
+
+Meta:
+    @epic vividus-engine
+    @feature aliases
+
+Scenario: Verify aliases loaded from resources
+
+!-- Vividus step
+Then `1` matches `\d+`
```

Comment:
is it possible to use a different language as an alias? f

Reply:
some adjustment will be required
https://jbehave.org/reference/latest/stories-in-your-language.html

Reply:
Does it make sense to create an issue for this? @valfirst @uarlouski

Reply:
https://github.com/vividus-framework/vividus/issues/2498